changelog/fragments/1708732907-add-postrm-script-to-deb-and-rpm-packages.yaml

```
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
+
+# Change summary; a 80ish characters long description of the change.
+summary: add postrm script to deb and rpm packages
```

```suggestion
summary: Fix issue uninstalling deb/rpm so it completely stops the service properly.
```

These are user facing so best to explain what it means to the user.